src/renderer/views/preload/PreloadProgressView.tsx

```diff
@@ -44,12 +44,12 @@ const PreloadProgressView = observer(() => {
   };
 
   const getCurrentStepMessage = () => {
-    if (currentStep < 1 || currentStep >= statusMessage.length) {
+    if (currentStep < 1 || currentStep > statusMessage.length) {
```

Comment:
It's more natural for me.
```suggestion
    if (!(0 < currentStep && currentStep <= statusMessage.length)) {
```